library/src/main/java/com/novoda/downloadmanager/DownloadManagerBuilder.java

```diff
@@ -12,6 +19,8 @@
  */
 public class DownloadManagerBuilder {
 
+    private static final int ANDROID_N = 24;
```

Comment:
I don't think either comment is against updates; besides Paul's, Ataul is saying to remove the target SDK and leave that to the library users (you can't remove the `minSdkVersion` tho).